app/components/other_scores_component.rb

```diff
@@ -27,7 +27,7 @@ def descriptions
   def guarantee_list
     {
       term: content_tag(:p, "Customer commitments"),
-      description: content_tag(:p, guarantee_list_render)
```

Comment:
This makes sense, a couple of questions for my own understanding: will the customer commitment description always be wrapped in paragraphs coming from Contentful?
Will there ever be a case where we need to `render_with_breaks`?

Reply:
Good question! Currently this is the only field that is just pasted into Contentful without any change to the formatting from the original csv file, and which we then `render_without_breaks`. Other fields, e.g. opening hours, are manually formatted when they are pasted into Contentful, and then we `render_with_breaks`. Provided those fields continue to not be manually formatted on entry to Contentful then we shouldn't need to `render_with_breaks` - are you wondering if we need a conditional to check whether they have been manually formatted?

For additional info: although we currently manually format the other text fields when we enter them to Contentful, that is likely to change. The AbilityNet accessibility review highlighted a problem with malformed descriptions lists, where `descriptions term` and `descriptions description` weren't being used as key-value pairs, and so I think that we will need to change the way that those pieces of data are entered in Contentful, as we will need to be able to separate the strings using `:` and `,` as separators.

Reply:
> are you wondering if we need a conditional to check whether they have been manually formatted

Oh I didn't even know that there are different format options 😅. Only reason I asked was to make sure that the formatting will always be the same and so removing the paragraph wrapper and changing the renderer to one without breaks wouldn't affect the visual if for some reason there is content that isn't a paragraph.